docs/books/admin_guide/06-users.md

```diff
@@ -661,44 +661,71 @@ newgrp [secondarygroups]
+Shell > groupadd groupA ; groupadd groupB 
+Shell > usermod -G groupA,groupB test1
+Shell > id test1
+uid=1000(test1) gid=1000(test1) groups=1000(test1),1001(groupA),1002(groupB)
+Shell > echo $SHLVL ; echo $BASH_SUBSHELL
+1
+0
+
+Shell > su - test1
+Shell > touch a.txt
+Shell > ll
+-rw-rw-r-- 1 test1 test1 0 10月  7 14:02 a.txt
+Shell > echo $SHLVL ; echo $BASH_SUBSHELL
+1
+0
+
+# Generate a new child shell
+Shell > newgrp groupA
+Shell > touch b.txt
+Shell > ll
+-rw-rw-r-- 1 test1 test1  0 10月  7 14:02 a.txt
+-rw-r--r-- 1 test1 groupA 0 10月  7 14:02 b.txt
+Shell > echo $SHLVL ; echo $BASH_SUBSHELL
+2
+0
+
+# You can exit the child shell using the `exit` command
+Shell > exit
+Shell > logout
+Shell > whoami
+root
 ```
 
-!!! Note
-
-    After using this command, the files will be created with the `GID` of its subgroup.
-
-The command `newgrp` without parameters reassigns the primary group.
-
 ## Securing
 
 ### `passwd` command
 
 The `passwd` command is used to manage a password.
+
 ```
 passwd [-d] [-l] [-S] [-u] [login]
 ```
+
 Examples:
+
 ```
-$ sudo passwd -l albert
-$ sudo passwd -n 60 -x 90 -w 80 -i 10 patrick
+Shell > passwd -l albert
+Shell > passwd -n 60 -x 90 -w 80 -i 10 patrick
 ```
 
 | Option | Description                                            |
 | ------ | ------------------------------------------------------ |
-| `-d` | Removes the password.                                    |
-| `-l` | Locks the account.                                       |
-| `-S` | Displays the account status.                             |
-| `-u` | Unlocks the account.                                     |
-| `-e` | Expires the password.                                    |
-| `-n days` | Minimum password lifetime.                          |
-| `-x days` | Maximum password lifetime.                          |
-| `-w days` | Warning time before expiration.                     |
-| `-i days` | Delay before deactivation when the password expires.|
-
-With the `passwd` command, locking an account is accomplished by adding `!!` before the password in the `/etc/shadow` file.
-
-Using the command `usermod -U` command only removes one of the `!`. So, the account remains locked.
+| `-d` | Permanently removes the password. For root (uid=0) use only.         |
+| `-l` | Permanently lock user account. For root (uid=0) use only.            |
+| `-S` | Displays the account status. For root (uid=0) use only.  |
+| `-u` | Permanently unlocks user account. For root (uid=0) use only.          |
+| `-e` | Permanently expires the password. For root (uid=0) use only.         |
+| `-n DAYS` | Minimum password lifetime. Permanent change. For root (uid=0) use only. |
+| `-x DAYS` | Maximum password lifetime. Permanent change. For root (uid=0) use only. |
+| `-w DAYS` | Warning time before expiration. Permanent change. For root (uid=0) use only. |
+| `-i DAYS` | Delay before deactivation when the password expires. Permanent change. For root (uid=0) use only.|
+
```

Comment:
Since all of these commands are "For root (uid=0) only." Why not specify that ahead of the table and limit the number of words in the table itself? If you are trying to add clarity to the document, (which appears to be your goal) then this would help.